pixi.toml

```diff
@@ -110,7 +110,6 @@ causalml = "*"
 tabulate = "*"
 
 [feature.benchmark.tasks]
-postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
```

Comment:
The command is identical to the existing `postinstall` command. Its existence forces the user to pick an environment when running it rather than using the "default" one.